### DISCUSSION TITLE
[Unstable] Missions successmessage replaced by description

### DISCUSSION THREAD

**Discussion by AzureNocte** [FakeFishGames/Barotrauma#13737] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I noticed that in 1.4.1.0, when round ends it displays missions that completed with mission's description texts, which should be mission's successmessage texts.

### Reproduction steps

1. Start a round with missions
2. complete missions
3. it shows description of mission instead of successmessage

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

Other

### -

v1.4.1.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! [I created an issue ticket about this](https://github.com/FakeFishGames/Barotrauma/issues/13746), we'll look into it!
